Adaptors/S3/src/ArmoniK.Core.Adapters.S3.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.S3" Version="3.7.101.46" />
+    <PackageReference Include="CommunityToolkit.HighPerformance" Version="8.0.0" />
```

Comment:
I believe you do not use it.

Reply:
I still don't know, I'm testing it with my bench project.

Reply:
In the current implementation I finaly use it.
method comments are : Returns a <see cref="Stream"/> wrapping the contents of the given <see cref="ReadOnlyMemory{T}"/> of <see cref="byte"/> instance.
For the first implementation I think it is good to use it.
If we find better way to implement S3 read/write perfs in the bench part I will update it.